corehq/motech/repeaters/models.py

```diff
@@ -368,7 +368,7 @@ def register(self, payload):
             'doc_type': self.doc_type
         })
         repeat_record.save()
-        repeat_record.attempt_forward_now()
+        repeat_record.attempt_forward_now(fire_synchronously)
```

Comment:
Another option here would be to make `fire_synchronously` an attr of the `Repeater` subclass.  Then that behavior would be configured there rather than in the signal receivers.  Both options seem plenty valid, just wanted to throw that out there.